accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -112,10 +123,310 @@ Windows-specific:
 
 * **DpapiProtectedConfigurationProvider**
 
+### System.Data.SqlTypes
+
+* **SqlFileStream**
```

Comment:
Should we mark the entire type?

Reply:
@David-Engel does this look correct to you?

Reply:
Mark the entire type. It's useless without `ntdll.dll`

Reply:
Looks correct to me.